app.json

```diff
@@ -46,6 +46,16 @@
             "description": "Name of your Heroku app, given in the first blank on this page.",
             "value": "",
             "required": true
+        },
+        "LOG_CHANNEL": {
```

Comment:
Dude, don't you dare..
Ultroid auto creates these stuff...

Reply:
if anyone not wanted to auto create these stufff so,

Reply:
Then they can do it by deploying, then adding these stuff to the config vars and restart.
It's best to keep this repo as minimal as possible.

Reply:
ok